jscomp/syntax/tests/ppx/react/expected/mangleKeyword.res.txt

```diff
@@ -1,21 +1,38 @@
 @@jsxConfig({version: 3})
 
-module C30 = {
-  @obj external makeProps: (~_open: 'T_open, ~key: string=?, unit) => {"_open": 'T_open} = ""
+module C3A0 = {
```

Comment:
It's not affected in the PR

Reply:
Had the same issue in my PR, here is the fix: https://github.com/rescript-lang/rescript-compiler/pull/6411/commits/ba56ffce0a6ba41b21b3afc01db23cb46e43a4f6